test/norm/contract_test.exs

```diff
@@ -39,7 +39,10 @@ defmodule Norm.ContractTest do
   end
 
   test "bad contract" do
-    assert_raise ArgumentError, ~r/got: `@contract\(foo\(n\)\)`/, fn ->
+    expected = if version().minor >= 13,
+      do: ~r/got: `@contract foo\(n\)`/, else: ~r/got: `@contract\(foo\(n\)\)`/
```

Comment:
I found this pattern while inspecting `test/norm/core/selection_test.exs`, so I decided to follow that approach to keep compatibility with previous versions.